packages/zowe-explorer/src/configuration/Profiles.ts

```diff
@@ -529,13 +529,14 @@ export class Profiles extends ProfilesCache {
 
     public async promptCredentials(profile: string | imperative.IProfileLoaded, rePrompt?: boolean): Promise<string[]> {
         ZoweLogger.trace("Profiles.promptCredentials called.");
+        const profilename = typeof profile === "string" ? profile : profile.name;
         const userInputBoxOptions: vscode.InputBoxOptions = {
             placeHolder: vscode.l10n.t(`User Name`),
-            prompt: vscode.l10n.t(`Enter the user name for the connection. Leave blank to not store.`),
+            prompt: vscode.l10n.t(`Enter the user name for the {0} connection. Leave blank to not store.`, profilename),
         };
         const passwordInputBoxOptions: vscode.InputBoxOptions = {
             placeHolder: vscode.l10n.t(`Password`),
-            prompt: vscode.l10n.t(`Enter the password for the connection. Leave blank to not store.`),
+            prompt: vscode.l10n.t(`Enter the password for the {0} connection. Leave blank to not store.`, profilename),
```

Comment:
For localization purposes, it would be best to add a comment for the `l10n` string so it can be more easily translated, e.g.:

```typescript
vscode.l10n.t({
    message: "Enter the password for the {0} connection. Leave blank to not store.",
    args: [profilename],
    comment: ["Profile name"]
});
```

Reply:
Addressed :)